homeassistant/components/kostal_plenticore/const.py

```diff
@@ -174,7 +223,12 @@
         "scb:statistic:EnergyFlow",
         "Statistic:Autarky:Total",
         "Autarky Total",
-        {ATTR_UNIT_OF_MEASUREMENT: PERCENTAGE, ATTR_ICON: "mdi:chart-donut"},
+        {
+            ATTR_UNIT_OF_MEASUREMENT: PERCENTAGE,
+            ATTR_ICON: "mdi:chart-donut",
+            ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
+            ATTR_LAST_RESET: LAST_RESET_NEVER,
```

Comment:
Is this really an accumulating sensor measurement? What does this measurement represent?

Reply:
Oh, sorry, I see that the Unit is Percentage which wouldn't make too much sense...

Reply:
Fixed.